### ISSUE TITLE
Alignment of axes between multiple plots

### ISSUE THREAD

**Issue by eaburns** [gonum/plot#151]
Original [issue 124](https://code.google.com/p/plotinum/issues/detail?id=124) created by eaburns on 2013-05-01T13:26:55.000Z:

I have two plots with different y axes but with the same x axis. I want to display one plot above the other. The problem is that the spacing differs slightly so the axes aren't perfectly aligned.

I don't see any way to solve this with the existing interface.

One way to solve it is to add an option to set the size of an axis to a fixed value. Basically, add a variable like this to the Axis struct:

```
// If nonzero, the axis will occupy a fixed width regardless of the
// content. This is useful if you need the axes to align between
// multiple plots.
FixedSize vg.Length
```

Then you can set the same FixedSize value for each of your plots and they will always align.

I've attached a patch which does this and solves the issue for me.

**Comment by brunoqc** [comment]
@eaburns by the way you can use "```"  around your code to have nice monospace formatting and syntax highlighting if you specify the "go" keyword.

    ```go
    <your code>
    ```

will render as:

```go
/*
Build a bunch of plots and align their x axes.
*/
var plots []plot.Plot
// build plots
for _, p := range plots {
    // AlignX aligns p's x axis with the other plots x axes.
    p.AlignX(plots...)
}
```

**Comment by eaburns** [comment]
Yeah, sorry, I often reply from my cell phone and don't think about that.

**Comment by ctessum** [comment]
Closed via #354
